packages/flame_bloc/example/analysis_options.yaml

```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```

Comment:
we should copy this file from the other repos to make sure things are linted properly

Reply:
*other projects. on this repo